cmake/projects/OpenSceneGraph/hunter.cmake

```diff
@@ -0,0 +1,24 @@
+# Copyright (c) 2016-2018, Ruslan Baratov
+# All rights reserved.
+
+# !!! DO NOT PLACE HEADER GUARDS HERE !!!
+
+include(hunter_add_version)
+include(hunter_cacheable)
+include(hunter_download)
+include(hunter_pick_scheme)
+
+hunter_add_version(
+    PACKAGE_NAME
+    OpenSceneGraph
+    VERSION
+    3.6.3-p0
+    URL
+    "https://github.com/t0p4/OpenSceneGraph/archive/v3.6.3-p0.tar.gz"
```

Comment:
We have talked about it at least 6 times:
* https://github.com/ruslo/hunter/pull/1670#discussion_r241994878
* https://github.com/ruslo/hunter/pull/1631#discussion_r236286364
* https://github.com/ruslo/hunter/pull/1626#discussion_r235530554
* https://github.com/ruslo/hunter/pull/1617#discussion_r232957871
* https://github.com/ruslo/hunter/pull/1596#discussion_r228043064
* https://github.com/ruslo/hunter/pull/1574#discussion_r223204344

Really?